docs/mir-snaps.md

```diff
@@ -0,0 +1,137 @@
+use as a template to create their own. If you are experimenting and simply
+building both snaps at the same time you should experience no problems,
+however once the `mir-server` snap is in the store, it is important that the
+mir-client use the `libmirclient` version corresponding to the `libmirserver`
+within their snap.
+
+The `mir-snap` is only targeted for Ubuntu Core. It is intended for
+kiosk-like products and applications. If products require many separate UI
+applications, then they need Ubuntu Personal.
+
+This document targets Xenial Ubuntu Core and assumes your host machine is
+running Xenial (up to date) and hence using Snapcraft 2.0.
+
+> *NOTE: 2016-02-03*
+> - snapcraft 2.1 came out today and requires changes. Will followup with
+>   updates.
+> - also, the most recent Xenial kernel has [a bug creating issues for
+>   mir](https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1540731)
+
+## Instructions
+
+You need to be on a Xenial host to obtain the proper tools and the latest
+versions of stage packages. Note, one could also use `lxc` for building snaps
+in a clean environment. Also, make sure Virtual Machine Manager is installed,
+from Ubuntu Software Center or
+
+```
+apt-get install virt-manager
+```
+
+### Get the Mir Snap running first
+
+If you haven't already, [install the snapcraft tools](get-started.md).
+
+Now grab the `mir-server` snap branch to build.
+```
+$ bzr branch lp:~mir-team/+junk/mir-server-snap/
+$ cd mir-server-snap
+$ snapcraft snap
+```
+Now to create an image off the rolling edge of ubuntu core development. You
+will need to do this to get the latest security capabilities for
+`display-server` and `mir-client`, do not use an older image.
+
+Also, you will need to use a "special" `ubuntu-device-flash` which is not
+quite released to xenial yet, download it from here
+https://people.canonical.com/~mvo/all-snaps/
+
+```
+$ sudo /dir/to/your/downloaded/ubuntu-device-flash --verbose core rolling --channel edge -o xenial_core_amd64.img --developer-mode --gadget=canonical-pc.canonical --kernel=canonical-pc-linux.canonical --os=ubuntu-core.canonical --size=5
+```
+
+Now launch your Virtual Machine Manager application (can be found & installed
+from the Ubuntu Software Center). Select the icon for "New Virtual Machine"
+(or menu to "File", "New Virtual Machine"). In the dialog select the radio
+button for "Import Existing Disk Image", browse to your `*.img` and select
+it. You can leave the other defaults. Once you select your way forward, it
+should launch another window with your Snappy VM in it. Once the system
+settles it will provide you a prompt. Just click on the window to interact
+with the VM, to escape select "control+alt" keys. To login, the Snappy image
+user is "ubuntu" and password is "ubuntu". Check your ip with ipconfig.
+
+[vm-image]: https://raw.githubusercontent.com/ubuntu-core/snapcraft/master/docs/images/ubuntucore_in_vmm.png
```

Comment:
Doesn't this require a !? Referencing the example here: https://guides.github.com/features/mastering-markdown/